### ISSUE TITLE
POST Data is not passing (iOS)

### ISSUE THREAD

**Issue by areanet-schmid** [capacitor-community/http#251]
I'm trying to do a simple post request on iOS.

```
const options = {
      url: SERVER_URL,
      headers: { 'Content-Type': 'application/json'  },
      data: { foo: 'bar', cool: true },
    };

Http.post(options);
```

SERVER_URL is a simple PHP script with:
`var_dump($_POST);`

POST data is not sending - see console log:
<img width="1015" alt="Bildschirmfoto 2022-04-13 um 11 43 25" src="https://user-images.githubusercontent.com/4621231/163148702-b9b5b1b8-669d-4c7d-9d35-13bd9ab2f5b2.png">

**Environment:**

- Ionic CLI 6.19.0 
- Capacitor CLI  3.4.3
- @ionic/angular 6.0.14
- @capacitor/ios 3.4.3
- @capacitor-community/http 1.4.1

**similar tickets**
https://github.com/capacitor-community/http/issues/208
https://github.com/capacitor-community/http/issues/67

**Comment by nhh** [comment]
We are experiencing this too. +1

**Comment by billpull** [comment]
@areanet-schmid are you only experiencing this on certain versions of iOS? I am testing a new app and users with iOS v14 are reporting issues that seem related.

**Comment by Gladiatornoshield** [comment]
Hi Guys any solution to this. I cant seems to get POST data on the server. But i can get data when using Postman. Any help would be grateful

This is the final item on my app and i really need a solution ASAP any help suggestions would be appreciated 

![image](https://user-images.githubusercontent.com/73953775/170093463-89489cec-1f57-48aa-9e2e-089b9ac16fc9.png)

**Comment by simonthesurgery** [comment]
I've also encountered this issue. 

Works fine in Chrome, but nothing is passed on iOS
<img width="2050" alt="Screenshot 2022-05-31 at 10 22 46" src="https://user-images.githubusercontent.com/26874246/171140855-3b2ad1ad-1a88-48e2-86dc-c2c5ad4fdb28.png">

iOS 15.5
capacitor/core 3.4.3
capacitor/ios 3.4.3
capacitor-community/http 1.4.1
ionic/angular  6.0.0

**Comment by Gladiatornoshield** [comment]
I ended up using axios until I find a permanent solutions. Works well on both iOS and Android

**Comment by gpereira13** [comment]
I fixed this by setting the 'Content-Type' header to my request as reading the ios plugging relies in the content type to parse the body data. 

See: 
https://github.com/capacitor-community/http/blob/a9103fecaba4d3d3b731b7f0fa7e939ec0c98b7f/ios/Plugin/CapacitorUrlRequest.swift#L46

maybe we could specify that on the documentation 

EDIT:
tested on IOS only

**Comment by mightytyphoon** [comment]
yeah the problem is known and I think pull requests were sent but there is absolutely no review by the maintainers. Create a http service with all requests inside and use axios, if one day capacitor http is finally working properly, you can add it by simply changing the http service and you won't have to change every requests in your app.
